### PR TITLE
Updated Past Events Font-Color to Primary for Visibility

### PR DESCRIPTION
**Pull request by Shyesta**
Issue: [#5164](https://github.com/twentyhq/twenty/issues/5164)

Updated font-color of the title of past calendar events to be primary to improve visibility. Calendar event time font-color remains unchanged.

Before:
image

After:
image

The year in month-year at the top of these screenshots also appears to be in tertiary font-color which can be adjusted too if that feels not visible enough.

### CODE REVIEW

**Comment by charlesBochet** [comment]
Hi @Shyesta, thanks for your contribution :)
1) When creating a PR, could you create branch in your fork instead of working from main, this will allow us to also push to your branch (we cannot push to your main branch
2) Is this tied to any issue, if yes, could you link it? Otherwise, we would need to get design team have a look

**Comment by Shyesta** [comment]
Oh! Sorry about that @charlesBochet. Forgot to include the issue in the comment. I went ahead and updated the original description to include a link to the issue.

As far as branching goes, do you know if I am able to update this PR to work off of a branch instead of main? Or do I need to open a new PR with the branch selected? Either way, I'm happy to fix that however I can.

File: packages/twenty-front/src/modules/activities/calendar/components/CalendarEventRow.tsx [hunk 1]
```diff
@@ -48,14 +48,15 @@ const StyledAttendanceIndicator = styled.div<{ active?: boolean }>`
 const StyledLabels = styled.div`
   align-items: center;
   display: flex;
-  color: ${({ theme }) => theme.font.color.tertiary};
+  color: ${({ theme }) => theme.font.color.primary};
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/activities/calendar/components/CalendarEventRow.tsx:51]: based on the issue, it seems that we want secondary and not primary?

**Comment by charlesBochet** [comment]
Ok, thank you! I've left a comment!

File: packages/twenty-front/src/modules/activities/calendar/components/CalendarEventRow.tsx (hunk 1 above)
Comment (Shyesta) [reply packages/twenty-front/src/modules/activities/calendar/components/CalendarEventRow.tsx:51]: I had a back and forth conversation with another member of Twenty in the comments of the original issue who said that primary looked better from the screenshots I posted.

Comment (charlesBochet) [reply packages/twenty-front/src/modules/activities/calendar/components/CalendarEventRow.tsx:51]: Ok, I missed it, LGTM!

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you @Shyesta

**Pull request merged**